### PR TITLE
Cryptocurrency conversion: Fixes #2306

### LINKED ISSUE #2306: Limit Cryptocurrency IA triggering for general queries
Query: https://duckduckgo.com/?q=1+VPN

Actual: Cryptocurrency IA is triggered and displays rate incorrectly "Vpn%"

Expected: No IA triggered - limit query for Cryptocurrency IA to trigger on conversion to other cryptocurrecies only.

2015-11-25-02-1 vpn at duckduckgo

________
https://duck.co/ia/view/cryptocurrency

### PR DESCRIPTION
**Pull request by claytonspinner**
For issue #2306. This fixes the issue with the IA incorrectly displaying cryptocurrencies with spaces in their name.
Original:
4334c3a0-93a4-11e5-808d-8ea61276adc1
Fix:
issue2306-vpn

Also, this limits the triggering when the query is '1 <cryptocurrency>'. Only the top 10 currencies or currencies with 'coin' in the name will trigger. The top 10 currencies are based on market cap. from [coinmarketcap.com](http://coinmarketcap.com/currencies/).

The following queries trigger:
`'1 ppc' // triggers because peercoin is in the top 10`
`'1 etherium' // triggers because etherium is in the top 10`
`'1 namecoin' // triggers because 'coin' is in the cryptocurrency name`

The following queries do not trigger:
`'1 vpn'`
`'1 gig'`
`'1 dogeparty'`

Also, a space is now required between the number and the cryptocurrency name for queries like '4shared'.

Trigger:
`'4 shared'`
`'200 litecoin'`

Do not trigger:
`'4shared'`
`'200litecoin'`

Let me know if there's any additional changes that I should add. Thanks!

https://duck.co/ia/view/cryptocurrency

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by MrChrisW** [comment]
Thanks @claytonspinner ! Will be reviewed shortly.

File: lib/DDG/Spice/Cryptocurrency.pm [hunk 1]
```diff
@@ -48,6 +48,20 @@ my @excludedCurrencies = (
     'usd'
 );
 
+# Used to filter on queries of the form '1 <cryptocurrency>'
```
Comment (MrChrisW) [inline lib/DDG/Spice/Cryptocurrency.pm]: Can you add the source of the top currencies to this comment `coinmarketcap.com` for future reference.

**Comment by MrChrisW** [comment]
@claytonspinner Tested and working, LGTM :+1: 

@jagtalon @moollaza for review.

**Comment by moollaza** [comment]
Deploying to Beta...

File: lib/DDG/Spice/Cryptocurrency.pm (hunk 1 above)
Comment (claytonspinner) [inline lib/DDG/Spice/Cryptocurrency.pm]: @MrChrisW Yup, added.

**Comment by MrChrisW** [comment]
@moollaza Can we get this one merged? I see no issues after testing.

**Comment by moollaza** [comment]
@claytonspinner @MrChrisW Thanks! This LGTM :+1:

Apologies for the delay!

**Pull request merged**